src/scripts/site.js

```diff
@@ -650,33 +616,163 @@ var AppInfo = {};
-            itemShortcuts: componentsPath + "/shortcuts",
-            playQueueManager: componentsPath + "/playback/playqueuemanager",
-            autoPlayDetect: componentsPath + "/playback/autoplaydetect",
-            nowPlayingHelper: componentsPath + "/playback/nowplayinghelper",
-            pluginManager: componentsPath + "/pluginManager",
-            packageManager: componentsPath + "/packagemanager",
-            screensaverManager: componentsPath + "/screensavermanager"
+            medialibrarycreator: "components/medialibrarycreator/medialibrarycreator",
+            medialibraryeditor: "components/medialibraryeditor/medialibraryeditor",
+            nowPlayingHelper: "components/playback/nowplayinghelper",
+            packageManager: "components/packagemanager",
+            playlisteditor: "components/playlisteditor/playlisteditor",
+            playQueueManager: "components/playback/playqueuemanager",
+            pluginManager: "components/pluginManager",
+            qualityoptions: "components/qualityoptions",
+            screensaverManager: "components/screensavermanager",
+            visibleinviewport: "components/visibleinviewport",
+            filesystem: "components/filesystem",
+            shell: "components/shell",
+            apiclient: "libraries/apiclient/apiclient",
+            imageFetcher: "components/images/imageFetcher",
+            alert: "components/alert",
+            dialog: "components/dialog/dialog",
+            loading: "components/loading/loading",
+            "multi-download": "components/multidownload",
+            fileDownloader: "components/filedownloader",
+            localassetmanager: "libraries/apiclient/localassetmanager",
+            transfermanager: "libraries/apiclient/sync/transfermanager",
+            filerepository: "libraries/apiclient/sync/filerepository",
+            localsync: "libraries/apiclient/sync/localsync",
+            fnchecked: "legacy/fnchecked",
+            legacyDashboard: "legacy/dashboard",
+            legacySelectMenu: "legacy/selectmenu",
+            events: "libraries/apiclient/events",
+            credentialprovider: "libraries/apiclient/credentialprovider",
+            connectionManagerFactory: "libraries/apiclient/connectionmanager",
+            appStorage: "libraries/apiclient/appStorage",
+            serversync: "libraries/apiclient/sync/serversync",
+            multiserversync: "libraries/apiclient/sync/multiserversync",
+            mediasync: "libraries/apiclient/sync/mediasync",
+            itemrepository: "libraries/apiclient/sync/itemrepository",
+            useractionrepository: "libraries/apiclient/sync/useractionrepository",
+            page: "libraries/pagejs/page",
+            headroom: "components/headroom/headroom",
+            scroller: "components/scroller",
+            navdrawer: "components/navdrawer/navdrawer",
+            queryString: "libraries/query-string/index",
```

Comment:
queryString ??

Reply:
The module is already named queryString in master. This is a simple alias to show it where to find it.

Reply:
i thought u move this to node

Reply:
Indeed, I'm dumb and clearly in need of some sleep :p